Yoshi/Yoshi/YoshiActionResult.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  YoshiActionResult.swift
+//  Pods
+//
+//  Created by Christopher Jones on 2/8/16.
+//
```

Comment:
Not sure what you mean? However I realized the enum needed to be in its own file, so I moved it.